spec/takeaway_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let (:item){ double :item }
+
+  it 'should have a function to view the menu' do
+    takeaway = Takeaway.new
+    expect(takeaway).to respond_to(:menu)
```

Comment:
Since you have more sophisticated tests of this method actually functioning, I think you can delete the redundant test.